src/openrct2/world/Park.cpp

```diff
@@ -548,8 +548,8 @@ money16 Park::CalculateTotalRideValueForMoney() const
 uint32_t Park::CalculateSuggestedMaxGuests() const
 {
     uint32_t suggestedMaxGuests = 0;
+    uint32_t difficultGenerationBonus = 0;
 
-    // TODO combine the two ride loops
```

Comment:
I don't think it makes sense to combine the two loops the old code was easier to read imho.